db/migrate/20180101010108_lit_create_lit_incomming_localizations.rb

```diff
@@ -13,10 +15,7 @@ def up
 
       t.timestamps
     end
-    add_index :lit_incomming_localizations, :locale_id
```

Comment:
any reason why those indexes have been removed?

Reply:
Got ya. But now it won't add indexes if called from Rails 4 - could you change this?

Reply:
Sure. 
Maybe we can switch to something like more custom. e.g.
```
change_table :lit_incomming_localizations do |t|
...
end
...
...
add_column :lit_incomming_localizations, :locale_id, :integer
add_index : lit_incomming_localizations, :locale_id
```

so it will pass both in rails 4 and 5. Explicit calls to method. So we don't use magic like `t.references`
What do you think?

Reply:
I'd add those columns in `change_table` block as integers and then add indexes - so exactly as currently `incomming_id` column is defined an